src/main/java/VASSAL/build/module/Inventory.java

```diff
@@ -1377,11 +1377,12 @@ protected String toSortKey() {
        * @return true if arguments looks processable, false else
        */
       protected boolean argsOK(Object arg0, Object arg1) {
-        return (arg0 != null &&
-                arg1 != null &&
-                arg0 instanceof CounterNode &&
-                arg1 instanceof CounterNode);
-        }
+        return (
```

Comment:
This shouldn't have parentheses around it at all...

Reply:
Good point, and given enough time to analyze my IDE suggests some more changes in this area, I will add them to this PR.